packages/m2m-gateway/src/utilities/errorMappers.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable sonarjs/no-identical-functions */
+import { constants } from "http2";
+
+const { HTTP_STATUS_INTERNAL_SERVER_ERROR } = constants;
+
+export const emptyErrorMapper = (): number => HTTP_STATUS_INTERNAL_SERVER_ERROR;
```

Comment:
This could be imported from commons after #1749, could you define it there in the common errors in the same place where I defined it in #1749? So that after merge we should have no conflict